### DISCUSSION TITLE
Have the bottle service still active.

### DISCUSSION THREAD

**Discussion by titis14** [nicolargo/glances#2230] [Q&A] [answered]
Hello,
I installed Glances and Bottle to have the web interface.
Everything works fine when I do the command glances -w.
My request is how to keep the web service active even when I exit the terminal?
I need the service to run all the time to get me information about Home Assistant.
Thank you

**Comment by nicolargo** [comment, marked as answer]
On Linux with SystemD, you can follow this Wiki page: https://github.com/nicolargo/glances/wiki/Start-Glances-through-Systemd

**Comment by titis14** [comment]
Hi,
It's perfect thank you
